docs/platforms/apple/common/index.mdx

```diff
@@ -342,6 +342,14 @@ if (error) {
 }
 ```
 
+<PlatformSection supported={[ "apple.macos"]}>
+
+## Uncaught Exceptions
+
+The Sentry Apple SDK can't capture uncaught exceptions out of the box. Please visit <PlatformLink to="/usage/#capturing-uncaught-exceptions">capturing uncaught exceptions</PlatformLink> for more information.
```

Comment:
afaik only on macOS, right? I know we show this snippet only on the macos docs, but prob still good to add "on macOS"